<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>700</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QTextEdit" name="ideTextEdit">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>660</width>
       <height>460</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="runPushButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>480</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="appOutputTextEdit">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>530</y>
       <width>661</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="AppOutputLabel">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>510</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Application Output</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="metadataTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>510</width>
       <height>240</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="subirImagenPushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>640</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Subir Imagen/Galer√≠a</string>
     </property>
    </widget>
    <widget class="QPushButton" name="visualizarImagenPushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualizar imagen seleccionada</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rollbackPushButton">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>480</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Rollback</string>
     </property>
    </widget>
    <widget class="QPushButton" name="commitPushButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>480</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Commit</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
